practice1/pr1.html

```diff
@@ -37,30 +37,88 @@
-            <a href="">Shop spectacles</a>
+        <div>
+            <h3 class="secondary-header">Spectacles<br>designed for faces.<br>The average ones, the unaverage ones and the very unaverage ones too.</h3>
+                <a href="" class="caption-small shop-btn-container"><img src="img/Frame (2).svg" alt="Shop here" class="shop-btn" />Shop spectacles</a>
         </div>
         <div class="block-bradley img-text">
-            <h4 class="caption-small">Narrow bridge</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Narrow bridge</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 2.png" alt="Bradley" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Bradley</a>
+                <span>1 565 kr</span>
+            </div>
         </div>
         <div class="block-gifford img-text">
-            <h4 class="caption-small">Adjustable nose pads</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Adjustable nose pads</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 3.png" alt="Gifford" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Gifford</a>
+                <span>1 565 kr</span>
+            </div>
         </div>
         <div></div>
         <div class="block-woolf img-text">
-            <h4 class="caption-small">Low crest</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Low crest</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 4.png" alt="Woolf" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Woolf</a>
+                <span>1 565 kr</span>
+            </div>        
         </div>
         <div class="block-colonnade img-text">
-            <h4 class="caption-small">Low crest</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Low crest</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 5.png" alt="Colonnade" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Colonnade</a>
+                <span>1 565 kr</span>
+            </div>        
+        </div>
+    </section>
+    <hr>
+    <h2 class="primary-header">Here's a beautiful fact. Your eyes have ten layers. You see through so much.</h2>
+    <hr>
+    <section class="tax-return-blocks">
+        <div class="tax-return-container">
```

Comment:
Nice class names 😎 
I would probably change the first one to `tax-return-section` tho